string.rst

```diff
@@ -629,6 +637,10 @@ You can convert emojis to short codes with the ``emoji-text`` locale::
     $transliterator->transliterate('Breakfast with 🥝 or 🥛');
     // => 'Breakfast with :kiwifruit: or :milk-glass:
 
+.. versionadded:: 7.1
+
+    The ``text-emoji`` and ``emoji-text`` locales were introduced in Symfony 7.1.
+
```

Comment:
Emoji component was created in 7.1. So versionnadded is not needed

Reply:
You are right! Fixed. Thanks.